pkg/awsvpc/examples/out/analysis_out/config_object_all_vpcs_.txt

```diff
@@ -0,0 +1,6 @@
+Endpoint connectivity for VPC 
```

Comment:
for which VPC is this endpoint connectivity given? missing in title

Reply:
I assume the connectivity report is empty because this VPC has no nodes?

Reply:
because no name field in aws, it might be in tags.. will leave this for another pr, what do you think?

Reply:
> I assume the connectivity report is empty because this VPC has no nodes?

yes both instances in vpcid:5

Reply:
> because no name field in aws, it might be in tags.. will leave this for another pr, what do you think?

currently the output is not readable this way, can at least change the title so that it is based on VPC UID and not only VPC name?